docker/Centos.Dockerfile

```diff
@@ -1,6 +1,11 @@
-FROM centos:7
+FROM centos:8
 
+RUN cd /etc/yum.repos.d/
+RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-*
```

Comment:
Can leave a comment explaining the `sed`

Reply:
Done

Reply:
but why do we need to comment all the occurrences of the mirrorlist? :)

Reply:
When adding support for sandbox rpm build, we were facing below error when running dnf commands,

dnf.exceptions.RepoError: Failed to download metadata for repo 'appstream': Cannot prepare internal mirrorlist: No URLs in mirrorlist
2022-01-31T15:28:05+0000 CRITICAL Error: Failed to download metadata for repo 'appstream': Cannot prepare internal mirrorlist: No URLs in mirrorlist

We followed https://forketyfork.medium.com/centos-8-no-urls-in-mirrorlist-error-3f87c3466faa and made the suggested changes to comment mirrorlist and change baseurl,

sudo sed -i -e "s|mirrorlist=|#mirrorlist=|g" /etc/yum.repos.d/CentOS-*
sudo sed -i -e "s|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g" /etc/yum.repos.d/CentOS-*